README.md

```diff
@@ -107,6 +107,8 @@ options, see [the Marathon docs](https://mesosphere.github.io/marathon/docs/).
 
 See [the documentation](https://mesosphere.github.io/marathon/docs/developing-vm.html) on how to run Marathon locally inside a virtual machine.
 
+Local Debug parameter `./bin/start --master local --zk zk://localhost:2181/marathon --jvm-debug 9999`
```

Comment:
We should not add this information to the global README.
I would suggest something under the development section, e.g.: https://mesosphere.github.io/marathon/docs/contributing.html